guard/src/commands/validate.rs

```diff
@@ -246,7 +256,40 @@ or rules files.
             alpabetical
         };
 
-        let empty_path = Path::new("");
+        let summary_type: BitFlags<SummaryType> =
+            app.get_many::<String>(SHOW_SUMMARY.0)
+                .map_or(SummaryType::FAIL.into(), |v| {
+                    v.fold(BitFlags::empty(), |mut st, elem| {
+                        match elem.as_str() {
+                            "pass" => st.insert(SummaryType::PASS),
+                            "fail" => st.insert(SummaryType::FAIL),
+                            "skip" => st.insert(SummaryType::SKIP),
+                            "none" => return BitFlags::empty(),
+                            "all" => {
+                                st.insert(SummaryType::PASS | SummaryType::FAIL | SummaryType::SKIP)
+                            }
+                            _ => unreachable!(),
+                        };
+                        st
+                    })
+                });
+
+        let output_type = match app.get_one::<String>(OUTPUT_FORMAT.0) {
+            Some(o) => OutputFormatType::from(o.as_str()),
+            None => OutputFormatType::SingleLineSummary,
+        };
+
+        let structured = app.get_flag(STRUCTURED.0);
+        if structured && !summary_type.is_empty() {
```

Comment:
if `summary_type` is actually initialized to the value `none`, will `summary_type.is_empty()` still return `true`?

Reply:
Yes, the part of code that builds summary_type returns an empty vector for none